dev/user.clj

```diff
@@ -242,12 +253,35 @@
 
 
   ;; Query Peer
-  (start {:fdb-query-peer-servers "localhost:8090"
-          :fdb-api-port            8080
-          :fdb-mode                "query"})
+  (start {:fdb-api-port 8080
+          :fdb-mode "query"
+          :fdb-query-peer-servers "localhost:8090"
+          :fdb-group-servers "query-server@localhost:11002"
+          :fdb-group-this-server "query-server"
+          :fdb-storage-type "memory"
+          :fdb-consensus-type "in-memory"})
 
   (stop)
 
+  (def ledger-peer (start-one {:fdb-api-port 8090
+                               :fdb-mode "ledger"
+                               :fdb-group-servers "ledger-server@localhost:11001"
+                               :fdb-group-this-server "ledger-server"
+                               :fdb-group-log-directory "./build/data/group"
+                               :fdb-storage-file-root "./build/data"
+                               #_#_:fdb-storage-type "memory"
+                               #_#_:fdb-consensus-type "in-memory"}))
```

Comment:
Do we still want those last two?

Reply:
I found it handy to be able to switch back-and-forth between in-memory and persisted. Usually I'm against leaving commented out code like this (that's what git is for), but in the `dev/user` ns it seemed more convenient. I'm happy to drop it you think we ought, though.

Reply:
Totally fine in dev/user